Jenkinsfile

```diff
@@ -0,0 +1,50 @@
+// url: https://github.com/INWT/inwt-templates/blob/master/jenkins/r-ci.Jenkinsfile
+
+pipeline {
+  agent { label 'docker1' }
+  options { disableConcurrentBuilds() }
+  environment {
+    CUR_PROJ = 'dbtools'
+    CUR_PKG = 'dbtools'
+    CUR_PKG_FOLDER = '.'
+    INWT_REPO = 'inwt-vmdocker1.inwt.de:8081'
+  }
+  stages {
+    stage('Launch mysql test database') {
+      steps {
+        sh '''
+          docker stop mysql-test-database || :
+          docker build -t mysql-test-database inst/db/mysql && docker run --name mysql-test-database -p 3301:3306 -d --rm mysql-test-database
```

Comment:
Ich hatte hier https://github.com/INWT/sports-shop-rating-db/blob/master/Jenkinsfile#L24 etwas geändert:
`--rm` flag beim Starten des Containers löschen, dann warten und ggf. logs anzeigen. So bekommt man mit, wenn beim Starten der Datenbank etwas schief geht. In die post Section kommt dann das Stoppen + Löschen des Containers.